### PR TITLE
feat(docs): Prettify with `prettier`

### PR DESCRIPTION
**Pull request by jackmu95**
## Description of Pull Request:

This PR adds support for prettifying markdown files with `prettier`.

## PR checklist:

**What kind of change does this PR introduce?** (check at least one)
- [ ] Bugfix
- [ ] Feature
- [x] Enhancement to an existing feature
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)
- [ ] Yes
- [x] No

If yes, please describe the impact:

**The PR fulfills these requirements:**
- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (i.e. `fixes #xxxx[,#xxxx]`, where "xxxx" is the issue number)
- [x] The PR should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] PR titles should following the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. "fix(alert): not alerting during SSR render", "docs(badge): Updated pill examples, fix typos", "chore: fix typo in docs", etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**
- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] The changes have not impacted the functionality of other components or directives 
- [ ] ARIA Accessibility has been taken into consideration (does it affect screen reader users or keyboard only users? clickable items should be in the tab index, etc)

**If adding a new feature, or changing the functionality of an existing feature, the PR's description above includes:**
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: docs/markdown/intro/README.md [hunk 1]
```diff
@@ -36,22 +37,21 @@ Or import Bootstrap and BootstrapVue `scss` files via a custom SCSS file:
 
 ```scss
 // custom.scss
-@import "node_modules/bootstrap/scss/bootstrap";
-@import "node_modules/bootstrap-vue/src/index.scss";
+@import 'node_modules/bootstrap/scss/bootstrap';
```
Comment (tmorehouse) [inline docs/markdown/intro/README.md]: Here as well, doesn't SCSS / SASS prefer double quotes over single quotes?

File: docs/markdown/intro/README.md [hunk 2]
```diff
@@ -102,8 +98,8 @@ $grid-breakpoints: (
 );
 
 // Then include the following
-@include "bootstrap/scss/bootstrap";
-@include "bootstrap-vue/src/index.scss";
+@include 'bootstrap/scss/bootstrap';
```
[the remark this answers is not in the document]
Comment (jackmu95) [reply docs/markdown/intro/README.md]: Both are valid. The trailing semicolon is required.

File: docs/markdown/intro/README.md (hunk 1 above)
Comment (jackmu95) [reply docs/markdown/intro/README.md]: See above.

File: docs/markdown/intro/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tmorehouse) [reply docs/markdown/intro/README.md]: OK, wasn't positive about that.

**Pull request merged**